### ISSUE TITLE
Screen Reader is not announcing any information for "Progress Bar" present on Common Control 1 page: A11y_WinForms.NETRuntime_CommonControl1_InfoAndRelationships.

### ISSUE THREAD

**Issue by shindepra** [dotnet/winforms#13656]
"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver."

### GitHub Tags
#A11ySev2; #A11yTCS; #WCAG1.3.1; #Win32; #DesktopApp; #Win11; #WinForms .NET Runtime; #BM_WinForms.NETRuntime_Win32_June2025; #Info and Relationships; #NVDA; #Screen Reader; #Element:ProgressBar;

### Environment Details
Application Name: WinForms Runtime
Microsoft Windows 11 Enterprise (24H2) Build on 26100.4061
Screen Reader used: NVDA
NVDA Version: NVDA version: 2025.3(2025.3.0.33433)

### Repro Steps

1. Launch the "Accessibility core app".
2. Main Form screen would appear.
3. TAB to "Common Control1" button and hit ENTER key.
4. TAB till "Progress Bar".
5. Observe that Screen Reader is not announcing any information for "Progress Bar" present on Common Control 1 page.

### Actual Result
Screen Reader is not announcing any information for "Progress Bar" present on Common Control 1 page.

### Expected Result
Screen Reader should announce the information for "Progress Bar" present on Common Control 1 page.

### User Impact
Screen Reader user will not be able understand the information about "Progress Bar" if screen reader is not announcing any information for "Progress Bar" present on Common Control 1 page

**Comment by LeafShi1** [comment]
[Progress Bar](https://learn.microsoft.com/en-us/windows/apps/design/controls/progress-controls) is a non-interactive control, so it cannot gain focus via the Tab key.

To make the Progress Bar control support screen readers, you can enhance accessibility by setting the accessibility properties or associating a label. Please refer to https://learn.microsoft.com/en-us/accessibility-tools-docs/items/win32/progressbar_name

**Comment by merriemcgaw** [comment]
@LeafShi1 can you or @Olina-Zhang  make the change to the test application to allow for better accessibility? Is there any way for us to do any announcements as the progress bar moves?

**Comment by LeafShi1** [comment]
> [@LeafShi1](https://github.com/LeafShi1) can you or [@Olina-Zhang](https://github.com/Olina-Zhang) make the change to the test application to allow for better accessibility? Is there any way for us to do any announcements as the progress bar moves?

Rewrite `ProgressBar` so that it can use Tab focus.  We will update the test application.

**Comment by Olina-Zhang** [comment]
@shindepra updated the app in same file share as above @LeafShi1 comment. Can you please uninstall that test app and install the updated one from shared path again on your machine for testing?

https://github.com/user-attachments/assets/18355c20-937e-47ed-b55c-dfded403aa8a
